lib/routes/gamer520/index.ts

```diff
@@ -0,0 +1,133 @@
+import { Data, DataItem, Route } from '@/types';
+import got from '@/utils/got';
+import { parseDate } from '@/utils/parse-date';
+import { Context } from 'hono';
+
+export const route: Route = {
+    path: '/:category?/:order?',
+    categories: ['game'],
+    example: '/gamer520/switchyouxi',
+    parameters: {
+        category: '分类，见下表',
+        order: '排序，发布日期: date; 修改日期: modified',
+    },
+    features: {
+        antiCrawler: true,
+    },
+    name: '文章',
+    maintainers: ['xzzpig'],
+    handler,
+    url: 'www.gamer520.com/',
+    description: `分类
+
+  | 所有 | Switch游戏下载 | 金手指 | 3A巨作 | switch主题 | PC游戏 |
+  | --- | --- | --- | --- | --- | --- |
+  | all | switchyouxi | jinshouzhi | 3ajuzuo | zhuti | pcgame |`,
+};
+
+// (await (await fetch('https://www.gamer520.com/wp-json/wp/v2/categories')).json()).map((category) => ({ slug: category.slug, id: category.id, name: category.name, link: category.link }));
+const CATEGORY_ARRAY = [
```

Comment:
> To adjust the no. of articles returned in the posts API, you can pass the `per_page` search parameter.

https://github.com/DIYgod/RSSHub/pull/14909#discussion_r1535799581

FYI, this applies to most of the WordPress API, including `/wp/v2/categories`